test/suite/test_rollback_to_stable19.py

```diff
@@ -0,0 +1,208 @@
+            self.skipTest('Prepare transactions for column store table is not yet supported')
+
+        # Create a table without logging.
+        uri = "table:rollback_to_stable19"
+        ds = SimpleDataSet(
+            self, uri, 0, key_format=self.key_format, value_format="S", config='log=(enabled=false)')
+        ds.populate()
+
+        # Pin oldest and stable to timestamp 10.
+        self.conn.set_timestamp('oldest_timestamp=' + timestamp_str(10) +
+            ',stable_timestamp=' + timestamp_str(10))
+
+        valuea = "aaaaa" * 100
+
+        # Perform several updates and removes.
+        s = self.conn.open_session()
+        cursor = s.open_cursor(uri)
+        s.begin_transaction()
+        for i in range(1, nrows + 1):
+            cursor[ds.key(i)] = valuea
+            cursor.set_key(i)
+            cursor.remove()
+        cursor.close()
+        s.prepare_transaction('prepare_timestamp=' + timestamp_str(20))
+
+        # Configure debug behavior on a cursor to evict the page positioned on when the reset API is used.
+        evict_cursor = self.session.open_cursor(uri, None, "debug=(release_evict)")
+
+        # Search for the key so we position our cursor on the page that we want to evict.
+        self.session.begin_transaction("ignore_prepare = true")
+        evict_cursor.set_key(1)
+        evict_cursor.search()
+        evict_cursor.reset()
+        evict_cursor.close()
+        self.session.commit_transaction()
+
+        # Pin stable timestamp to 20.
+        self.conn.set_timestamp('stable_timestamp=' + timestamp_str(20))
+        if not self.in_memory:
+            self.session.checkpoint()
+
+        if not self.in_memory:
+            if self.crash:
+                simulate_crash_restart(self, ".", "RESTART")
+            else:
+                # Close and reopen the connection
+                self.reopen_conn()
+        else:
+            self.conn.rollback_to_stable()
+            s.rollback_transaction()
+
+        # Verify data is not visible.
+        self.check(valuea, uri, 0, 20)
+        self.check(valuea, uri, 0, 30)
+
+        stat_cursor = self.session.open_cursor('statistics:', None, None)
+        upd_aborted = stat_cursor[stat.conn.txn_rts_upd_aborted][2]
+        keys_removed = stat_cursor[stat.conn.txn_rts_keys_removed][2]
+        self.assertGreater(upd_aborted, 0)
+        self.assertGreater(keys_removed, 0)
+
+    def test_rollback_to_stable_with_history(self):
+        nrows = 1000
+
+        # Prepare transactions for column store table is not yet supported.
+        if self.key_format == 'r':
+            self.skipTest('Prepare transactions for column store table is not yet supported')
+
+        # Create a table without logging.
+        uri = "table:rollback_to_stable19"
+        ds = SimpleDataSet(
+            self, uri, 0, key_format=self.key_format, value_format="S", config='log=(enabled=false)')
+        ds.populate()
+
+        # Pin oldest and stable to timestamp 10.
```

Comment:
Pin oldest and stable to timestamp 10. -> Pin oldest and stable timestamp to 10.

Reply:
Fixed.